app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PagesController < ApplicationController
   def now
+    @members = Member.members.members
```

Comment:
:rofl: 

As much as I love `Member.members.members`, could this be moved into Member` as a class method? It'd make the implementation a little clearer, and keep all the API stuff confined to that one class.

Reply:
haha I wanted to see if I could sneak that past you